src/blocks/Continue.js

```diff
@@ -0,0 +1,31 @@
+import Blockly from 'blockly';
+import { time } from './Color';
+
+const Continue = {
+  definition: {
+    init() {
+      this.appendDummyInput()
+        .appendField('continue');
+      this.appendValueInput('LENGTH')
+        .setCheck('Number');
+      this.appendDummyInput()
+        .appendField(new Blockly.FieldDropdown([['seconds', 'UNITS_seconds'], ['milliseconds', 'UNITS_milliseconds']]), 'UNITS');
+      this.setInputsInline(true);
+      this.setPreviousStatement(true);
+      this.setNextStatement(true);
+      this.setColour(time.HUE);
+      this.setTooltip('');
+      this.setHelpUrl('');
```

Comment:
Now it's going to be easier to edit these when we get to writing the docs for each block.